SU-HSDO/suhumsci
# feat(STN-430): Spotlight design update/enhancements

# READY FOR REVIEW

## Summary
Tweaks design for colorful and traditional themes on the Spotlight paragraph type.

**Browser Testing**

## Steps to Test
1. `npm run test` passes. Run: `lando drush @sparkbox_sandbox.local config-import --partial` to import the spotlight database changes.
2. Create a spotlight page with the basic page with left/right and tall/short variations. Check that the design looks good in humsci traditional and colorful.
3. Verify designs match the browser: https://www.figma.com/file/bG42fKKioEcnJ0ORKkLL5E/%E2%9A%99%EF%B8%8F-Traditional-Theme---Working?node-id=5686%3A2 and https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme---Working?node-id=5710%3A0

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
